airbyte-integrations/connectors/source-oracle/src/main/java/io/airbyte/integrations/source/oracle/OracleSource.java

```diff
@@ -15,11 +15,13 @@
 import io.airbyte.integrations.source.jdbc.AbstractJdbcSource;
 import io.airbyte.integrations.source.relationaldb.TableInfo;
 import io.airbyte.protocol.models.CommonField;
+
+import java.io.*;
```

Comment:
Please check [Airbyte code style](https://docs.airbyte.io/contributing-to-airbyte/code-style)

Reply:
done